pkg/cloudprovider/cloudapi/aws/aws_converters.go

```diff
@@ -114,42 +115,44 @@ func convertFromSecurityGroupPair(cloudGroups []*ec2.UserIdGroupPair, managedSGs
 
 // convertFromIPPermissionToIngressRule converts cloud ingress rules from ec2.IpPermission to internal securitygroup.IngressRule.
 // Each AT Sg can have one or more ANPs and an ANP can have one or more rules. Each rule can have a description.
-func convertFromIPPermissionToIngressRule(ipPermissions []*ec2.IpPermission, managedSGs map[string]*ec2.SecurityGroup,
-	unmanagedSGs map[string]*ec2.SecurityGroup) []securitygroup.IngressRule {
-	var ingressRules []securitygroup.IngressRule
+func convertFromIPPermissionToIngressRule(sgID securitygroup.CloudResourceID, ipPermissions []*ec2.IpPermission,
```

Comment:
nit: just pass the sgID string directly from the caller.

Reply:
Done